wLabs/LabSetup.md

```diff
@@ -0,0 +1,595 @@
+---
+title: Setting up the Lab Environment
+layout: custom
+keywords: [lab, guide, setup, start, FinCube, data cells]
+description: This page details the necessary requirements and steps to set up a local environment that can accommodate a user following the Lab Guide.
+---
+* * *
+
+## Overview
+
+The [Lab Guide](/wLabs/lab.html) provides a convenient way for users to become familiar with Interject. The Guide walks through several individual walkthroughs on how to modify and create reports, drilling to data, and exporting reports. In order for a user to follow along, it is necessary to first set up a local environment that can accommodate this Lab Guide. This page details the necessary requirements and steps to do this.
+
+### Interject Data Report Flow
+
+The reports in the Lab Guide contain [Interject report formulas](/wIndex/Excel-Function-Index.html), each referencing a [Data Portal](/wPortal/Data-Portals.html) setup on the [Interject Portal Website](/wPortal/The-INTERJECT-Website-Portal.html). These Data Portals in turn reference a Data Connection and a Stored Procedure that point to a data source where the data will be pulled from. When the data source is a database, the entire flow can be conceptualized by the following diagram:
+
+![](/images/LabSetup/FlowChart.png)
```

Comment:
This diagram is mostly right, but it does look abit like Interject talks to the client's database through the Interject platform. This isnt the case and often times in client\sales meetings ive noticed the question comes up about whether interject reaches into the database or not. Its a worthy point to provide clarification on for new users.

![image](https://github.com/GoInterject/GoInterject.github.io/assets/10929547/7b90ea0c-3951-4590-92bd-a7d41dbac1fe)